hoodoo-digital/acme
# Feature/classes in stories

closes #47 and closes #18 

## Details

- What I did
   - Update templates with instructions to create stories
   - Add functionality to add css classes directly to stories
   - Prevent overwrite of existing stories
- How to test 
   - Run `acme` scripts to generate stories and go through development flow to create new stories
- Does this change require an update to the documentation?
   - We should document the development process

Include answers to these questions and any other relevant details
